### ISSUE TITLE
Update documentation according to 3.0 changes

### ISSUE THREAD

**Issue by mmnosek** [Starcounter/authorization#67]
After releasing `3.0`, documentation has to be updated. There's a lot from ASP.NET, so we should link to this docs at right places.

**Comment by warpech** [comment]
I think this was fixed in 4d04e0a068cd99c9c09f451467892d1029d3d3a9, 7670f6114057352a31e8f807489eb774b9996fe0.

@mmnosek can you confirm?

**Comment by mmnosek** [comment]
Nope. Look at `master` branch. We need the same information in `3.x`
